packages/core/addon/models/request/base.ts

```diff
@@ -39,6 +39,9 @@ export default class Base<T extends ColumnType> extends Fragment.extend(Validati
   @attr('string')
   field!: string;
 
+  @attr('string')
+  alias?: string | null;
```

Comment:
Will this serialize in filters/sorts (e.g. `filter.alias = 'test'` and `filter.serialize()`)?

We probably can do
```suggestion
  declare alias?: string | null;
```
and then keep `@attr('string')` in `column.ts`